IPython/html/static/edit/js/editor.js

```diff
@@ -176,6 +192,10 @@ function($,
             }
             codemirror.setOption(opt, value);
         });
+        var that = this;
+        codemirror.on('changes', function(cm, cahnges){
```

Comment:
Also, this is probably not the right place to set this, since this method is called on any update to config (e.g. toggling line numbers). This should probably be called when the CodeMirror object is instantiated.